### PR TITLE
chore: update readme.md

### PR DESCRIPTION
**Pull request by danielwernerio**
**- What I did**

Improvement of the representation of NanoCl in context of the README.md.

**- How I did it**

My own personal thoughts.

**- How to verify it**

Please read it and give me feedback.

**- Description for the changelog**

Optimized readme.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,73 @@
...
+<blockquote>
+ <span>
+    Who said that K8s is more than a perfect platform for application workloads?
+ </span>
+</blockquote>
 
 ## Installation
 
-To install **Nanocl** for your system, please refer to our online [installation guide][nanocl_install_guide].
+We are already compatible with **Debian/Ubuntu, MacOS & Windows**, just jump to [Installation guide][nanocl_install_guide].<br/>
```
Comment (0xle0ne) [inline README.md]: Can you replace Debian/Ubuntu with linux ?

**Review by 0xle0ne — CHANGES_REQUESTED** [review]
A last thing before merging

**Review by 0xle0ne — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (danielwernerio) [reply README.md]: This has to be corrected within the documentation, too.

Comment (0xle0ne) [reply README.md]: Yeap will do during the day

Comment (danielwernerio) [reply README.md]: Perfect. Leo is a real hustler.

**Pull request merged**